src/docs/reference/modules/ContentFields/README.md

```diff
@@ -231,66 +183,36 @@ When adding the field to a content type, use the settings to specify whether to
     ```
 
 === "Razor"
-
-    ``` html
-    @model OrchardCore.ContentFields.ViewModels.DisplayUserPickerFieldViewModel
-    @using OrchardCore.Mvc.Utilities
-
+    ```html
     @{
-        var name = (Model.PartFieldDefinition.PartDefinition.Name + "-" + Model.PartFieldDefinition.Name).HtmlClassify();
         var users = await @Orchard.GetUsersByIdsAsync(Model.UserIds);
     }
-
-    <div class="field field-type-userpickerfield field-name-@name">
-        <span class="name">@Model.PartFieldDefinition.DisplayName()</span>
-        @if (users.Any())
-        {
-            foreach (var user in users)
-            {
-                <span class="value">@user.UserName</span>
-                if (user != users.Last())
-                {
-                    <span>,</span>
-                }
-            }
-        }
-        else
+    @foreach (var user in users)
+    {
+        <span class="value">@user.UserName</span>
+        @if (user != users.Last())
         {
-            <span class="value">@T["No users."]</span>
+            <span>,</span>
         }
-    </div>
-
+    }
     ```
 
 #### Video
-
 <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/vqXwK69vtMw" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
-### `MultiText Field`
-
+### `MultiTextField`
 #### Video
-
 <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/WfP_rXz1id0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
 ## Creating Custom Fields
 
-### What to extend
+### When to Extend
 
-Before creating a new field, the solution might be to provide a custom editor and formatter instead.
-
-A field should represent some specific physical data and logical data. The same field can be customized
-to be edited and rendered differently using both Editors and Formatters. Editors are shapes that can
-be used to edit a field differently, for instance the WYSIWYG HTML editor is a custom editor for the HTML
-field. Formatters are alternate shapes that can be used to render a field on the front end, for instance
-a `Link` field could be rendered as a Youtube video player.
+Consider creating custom editors/formatters before new fields. Fields should represent specific data types, while editors/formatters handle input/output variations.
```

Comment:
Formatters are not explained anymore. There's a bunch of omissions from this downwards too, but I won't point them out one by one, please just check.